src/main/java/com/liveramp/cascading_ext/flow/JobRecordListener.java

```diff
@@ -20,11 +25,13 @@
 
   private final boolean failOnCounterFetch;
   private final JobPersister persister;
+  private final List<TaskSummary> taskSummaries;
```

Comment:
that's not quite what I meant.  I mean, it should be null until it's actually populated in recordStepData and we can be sure it has all the values it is going ot have